File: easy/1. Two Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        hashmap = {}

        for i in range(len(nums)):
            complement = target - nums[i]

            if complement in hashmap:
                return [i, hashmap[complement]]

            hashmap[nums[i]] = i

        return []
        
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        

File: js/1. Two Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let max = nums.length;
    
    for (var i = 0; i < max; i++) {
        for ( var j = 0; j < max; j++) {
            if (nums[i] + nums[j] == target && i != j) {
                return [i, j];
            };
        }
    }
};

twoSum([3,2,4], 6);